Extension/package.json

```diff
@@ -42,6 +42,168 @@
+            "media": {
+              "altText": "markdown file with instructions for configuring default build task on Mac with Clang",
+              "path": "walkthrough/build-project-mac.md"
+            }
+          },
+          {
+            "id": "build.project.linux",
+            "title": "Build your C++ file",
+            "description": "Create a default build task for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-linux#_build-helloworldcpp)",
+            "when": "isLinux",
+            "media": {
+              "altText": "markdown file with instructions for configuring default build task on Linux with GCC",
+              "path": "walkthrough/build-project-linux.md"
+            }
+          },
+          {
+            "id": "build.project.windows",
+            "title": "Build your C++ file",
+            "description": "Create a default build task for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-msvc#_build-helloworldcpp)",
+            "when": "isWindows",
+            "media": {
+              "altText": "markdown file with instructions for configuring default build task on MSVC with Windows",
+              "path": "walkthrough/build-project-windows.md"
+            }
+          },
+          {
+            "id": "run.project.mac",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-clang-mac#_debug-helloworldcpp)",
+            "when": "isMac",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on macOS",
+              "path": "walkthrough/run-and-debug-project-mac.md"
+            }
+          },
+          {
+            "id": "run.project.linux",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-linux#_debug-helloworldcpp)",
+            "when": "isLinux",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on Linux",
+              "path": "walkthrough/run-and-debug-project-linux.md"
+            }
+          },
+          {
+            "id": "run.project.windows",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-msvc#_debug-helloworldcpp)",
+            "when": "isWindows",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on Windows",
+              "path": "walkthrough/run-and-debug-project-windows.md"
+            }
+          },
+          {
+            "id": "install.cmake.tools",
+            "title": "Install CMake Tools",
```

Comment:
I think we should keep "Install CMake Tools" as the title for now to diversify the titles of the tasks a bit, since the previous two start with "Build" and "Debug." Especially since this task doesn't actually tell users how to build or debug their CMake projects, it just tells them to install the extension. 

Down the road, we will only show the step about CMake Tools if the customer has a CMakeLists.txt in their project. And in that case, we'll hide the steps for building a C++ file and debugging a C++ file, because they wouldn't need to know how to create tasks.json and launch.json if they're using the CMake Tools extension. When we do that, I agree we should change the title to "Build and Debug your CMake project with CMake Tools" or something like that.

Reply:
You misunderstood my comment (GitHub might be placing them at the wrong spot). I was referring to the descripton text and not the title.

Reply:
Got it, sorry about that misunderstanding! GitHub is showing the id and title for this comment. I'll update the description to include "build and debug" not just "build" -- good idea

Reply:
Oh, I just noticed the missing "your," that's probably what you were referring to. Fixing that, too.